src/Exercise/BlorkemonCards.elm

```diff
@@ -0,0 +1,60 @@
+module Exercise.BlorkemonCards exposing (..)
+
+import Analyzer exposing (CalledFrom(..), CalledFunction(..), Find(..))
+import Comment exposing (Comment, CommentType(..))
+import Dict
+import Review.Rule exposing (Rule)
+import RuleConfig exposing (AnalyzerRule(..), RuleConfig)
+
+
+ruleConfig : RuleConfig
+ruleConfig =
+    { slug = Just "blorkemon-cards"
+    , restrictToFiles = Just [ "src/BlorkemonCards.elm" ]
+    , rules =
+        [ CustomRule maxPowerUsesMax
+        , CustomRule sortByMonsterNameUsesSortBy
+        , CustomRule expectedWinnerUsesCompareShinyPower
+        , CustomRule expectedWinnerUsesCase
+        ]
+    }
+
+
+maxPowerUsesMax : Rule
+maxPowerUsesMax =
+    Analyzer.functionCalls
+        { calledFrom = TopFunction "maxPower"
+        , findFunctions = [ FromExternalModule [ "Basics" ] "max" ]
+        , find = Some
+        , comment = Comment "maxPower doesn't use max" "elm.blorkemon-cards.use_max" Essential Dict.empty
```

Comment:
Actually to be honest, "maxPower doesn't use max" is not used anywhere, it's only there for the sake of humans explaining when this comment would pop up.
It makes tests and rules a little bit more readable, that's all. Maybe it's not worth it...

Reply:
I think I would prefer to remove these in that case, but probably wants a longer discussions and definitely a different pr. Maybe we could just make the elm.blorkemon-cards.use_max more descriptive?

Reply:
Yes, maybe you could open an issue?
We cannot touch the `elm.blorkemon-cards.use_max` field, because it follows Exercism specs. We could use mores descriptive file names, maybe, but I'm afraid of adding too much complexity there.